appveyor.yml

```diff
@@ -0,0 +1,32 @@
+version: 1.0.x.{build}
+platform:
+- x64
+clone_depth: 10
+init:
+- SET
+- 'mkdir C:\ProgramData\PuppetLabs\code && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\facter && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\hiera && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\puppet\var && exit 0'
+environment:
+  matrix:
+  - PUPPET_GEM_VERSION: 3.4.3
+    RUBY_VER: 193
+  - PUPPET_GEM_VERSION: 3.7.5
+    RUBY_VER: 193
+  - PUPPET_GEM_VERSION: 3.7.5
+    RUBY_VER: 200-x64
+  - PUPPET_GEM_VERSION: 4.0.0
+    RUBY_VER: 21
+  - PUPPET_GEM_VERSION: 4.0.0
+    RUBY_VER: 21-x64
```

Comment:
Test against the latest release of most minor versions, capturing the oldest, the newest and a release somewhere in between.